src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(18);
+        user.setLogin("noOneIsClownBoy");
+        user.setPassword("1337228");
+    }
+
+    @AfterEach
+    void storageClose() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void passwordLength_notOk() {
+        user.setPassword("smal");
```

Comment:
```suggestion
        user.setPassword("smal1");
```
давай перевіряти наближене до критичного значення